packages/component-submission/client/pages/EditorsStepPage.test.js

```diff
@@ -269,12 +269,12 @@ describe('EditorsPage component', () => {
       expect(
         wrapper
           .instance()
-          .calculateNewSuggestedReviewersFieldValue(mockReviewers),
+          .calculateNewSuggestedReviewersFieldValue(mockReviewers, 3, 0),
```

Comment:
Looks like magic numbers - what do they mean? Can we use the same consts as in the function?

Reply:
oops didn't realise this is a test... so not as important